src/pages/workspaces/workspace/WorkspaceMenu.test.js

```diff
@@ -43,7 +46,7 @@ const workspaceMenuProps = {
 }
 
 beforeEach(() => {
-  MenuTrigger.mockImplementation(({ content }) => { return div([content]) })
+  MenuTrigger.mockImplementation(({ content }) => { return div({ role: 'menu' }, [content]) })
```

Comment:
This is necessary so that the a11y tests don't fail due to menu items that do not have an appropriate parent container (caused by the mocking of `MenuTrigger`).